velox/core/PlanNode.h

```diff
@@ -816,6 +838,10 @@ class GroupIdNode : public PlanNode {
     return groupingSets_;
   }
 
+  const std::vector<std::vector<std::string>>& groupingSetsWithColumns() const {
```

Comment:
Is groupingSets() method used in Prestissimo? If not, let's reuse that name.

Reply:
Sure.